Author: Jari Aalto <jari.aalto@cante.net>
Description: Add hungarian support

Index: wmitime/wmitime/hungarian.h
===================================================================
--- /dev/null	1970-01-01 00:00:00.000000000 +0000
+++ wmitime/wmitime/hungarian.h	2007-03-26 12:16:58.000000000 +0300
@@ -0,0 +1,28 @@
+/* hungarian.h */
+
+static char daynames[7][3] =
+{
+    {"Va"},
+    {"He"},
+    {"Ke"},
+    {"Sz"},
+    {"Cs"},
+    {"Pe"},
+    {"Sz"}
+};
+
+static char monthnames[12][4] =
+{
+    {"Jan"},
+    {"Feb"},
+    {"Mar"},
+    {"Apr"},
+    {"Maj"},
+    {"Jun"},
+    {"Jul"},
+    {"Aug"},
+    {"Sze"},
+    {"Okt"},
+    {"Nov"},
+    {"Dec"}
+};
Index: wmitime/wmitime/wmitime.c
===================================================================
--- wmitime.orig/wmitime/wmitime.c	2007-03-26 12:17:28.000000000 +0300
+++ wmitime/wmitime/wmitime.c	2007-03-26 12:17:56.000000000 +0300
@@ -35,11 +35,19 @@
 #define fr
 #endif
 
+#ifdef hu_HU
+#define hu
+#endif
+
 #ifdef fr
 #include "french.h"
 #else
+#ifdef hu
+#include "hungarian.h"
+#else
 #include "language.h"
 #endif
+#endif
 
 #include "wmitime-master.xpm"
 char wmitime_mask_bits[64*64];
